bin/v-add-user-package

```diff
@@ -130,6 +130,10 @@ RATE_LIMIT='$RATE_LIMIT'
 DATABASES='$DATABASES'
 CRON_JOBS='$CRON_JOBS'
 DISK_QUOTA='$DISK_QUOTA'
+CPU_QUOTA='$CPU_QUOTA'
```

Comment:
We should include input validation for it

Current state is possible to break things / execute bad things